src/metabase/shared/util/time.cljc

```diff
@@ -76,8 +76,10 @@
 (defn format-unit
   "Formats a temporal-value (iso date/time string, int for hour/minute) given the temporal-bucketing unit.
    If unit is nil, formats the full date/time"
```

Comment:
It would be great if you explained how `locale` is used. In particular, it's not easy to intuit what happens if `locale` is not provided.